test/index.test.js

```diff
@@ -1,12 +1,146 @@
 /*global describe, it */
 
 var expect = require("chai").expect,
-	db = require("../index");
+	getConnectionOptions = require("./getConnectionOptions"),
+	Database = require("../index");
 
 describe("iw-mysql-wrapper", function () {
 
-	it("should be defined", function () {
-		expect(db).to.be.a("function");
+	describe("Database.paramify()", function () {
+
+		it("should return empty values/tokens when list is empty", function () {
+			var result = Database.paramify([], "pfx");
+			expect(result).to.eql({
+				values: {},
+				tokens: []
+			});
+		});
+
+		it("should create a values/tokens map for the list", function () {
+			var result = Database.paramify(["one", "two"], "pfx");
+			expect(result).to.eql({
+				values: {
+					"pfx0": "one",
+					"pfx1": "two"
+				},
+				tokens: [":pfx0", ":pfx1"]
+			});
+		});
+
+	});
+
+	describe("Database()", function () {
+
+		var db;
+		beforeEach(function () {
+			db = new Database(getConnectionOptions());
+		});
+
+		afterEach(function (done) {
+			db.end(done);
+		});
+
```

Comment:
Yes. Surely, you've seen them before in `gampee` (using `mocha` here as well, not `buster`)